pipeline.wdl

```diff
@@ -68,6 +69,36 @@ workflow pipeline {
                 variantCalling = variantCalling,
                 dockerImages = dockerImages
         }
+
+        if (variantCalling) {
+        # Preprocess BAM for variant calling
+            call preprocess.GatkPreprocess as preprocessing {
+                input:
+                    bamFile = sample.bam,
+                    outputDir = outputDir + "/",
```

Comment:
Shouldn't these BAM files end up in the sample's directory?

Reply:
Ideally yes. Apparantly there was no test for that. I will fix it.